packages/core/src/tracing/index.js

```diff
@@ -41,8 +41,8 @@ let processIdentityProvider = null;
 
 // Note: Also update initializedTooLateHeuristic.js and the accompanying test when adding instrumentations.
 let instrumentations = [
-  './instrumentation/cloud/aws-sdk/v2/index',
-  './instrumentation/cloud/aws-sdk/v3/index',
+  './instrumentation/cloud/aws-sdk/v2',
```

Comment:
As part of the updates made in https://github.com/instana/nodejs/pull/866, we've introduced an instrumentationName in our AWS SDK instrumentation. This enhancement allows us to disable the tracer effectively using the instrumentation name rather than relying on the instrumentation syntax.

Reply:
I agree that keeping the logic in general might be a good idea, because the path could vary.

Why do we have to keep `exports.instrumentationName = 'aws-sdk/v3';`?
This is already the correct name.

Reply:
Yes, so I removed the instrumentationName from aws sdk v2 and v3 and keep the core logic as it is.